mitiq/calibration/settings.py

```diff
@@ -139,9 +143,30 @@ class Strategy:
 
     @property
     def mitigation_function(self) -> Callable[..., float]:
-        return partial(
-            self.technique.mitigation_function, **self.technique_params
-        )
+        if self.technique is MitigationTechnique.PEC:
+            params_copy = deepcopy(self.technique_params)
+            rep_function = params_copy.pop("representation_function")
+            operations = params_copy.pop("operations")
+            noise_level = params_copy.pop("noise_level")
+            is_qubit_dependent = params_copy.pop("is_qubit_dependent", True)
+            representations = [
+                rep_function(
+                    Circuit(op(*LineQubit.range(2))),
```

Comment:
Here we hard code `LineQubit.range(2)`, this is fine only in the case in which `is_qubit_depedent` is `False`.

Maybe, instead of passing gates $[CNOT, CZ]$ to the `operations` argument, we could pass operations (gates on some qubits).

In this way we can get rid of `LineQubit.range(2)`, here.

Reply:
Good catch, I've fixed that.